main.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Kindly remove this, as it is decapitating linters from performing and a standard syntax check on your code.
Don't forget that organizations use this to enforce coding standards in their organization, and since your will be part of one soon, I think you should get used to mastering linting styles :smile: :muscle: